docs/self-managed/platform-deployment/docker.md

```diff
@@ -46,6 +46,10 @@ With the Camunda 8.2.0 release and onward, all Camunda 8 Docker images are provi
 Your Docker client should automatically pull the image that suits your platform.
 We currently only recommend the `linux/amd64` for production usage, as the `linux/arm64` image is provided mainly for development purposes.
 
+:::note
+For Web Modeler, we only provide multi-platform images from the following releases onward: 8.2.8, 8.3.1, 8.4.0-alpha1.
```

Comment:
Is the patch level specific or required here?

It seems odd that we introduce this feature-like experience in a patch. I recommend we add this to the Announcements page. We don't have to do that immediately, but if we recognize we need more visibility, we can add that.

Reply:
Hm, true, it's like a new feature. We thought it would make sense anyway to backport it, so that you'd also get the benefits of the ARM images for the 8.2 and 8.3 versions.

I mentioned the specific patch levels here to make it clearer that this statement is not exactly true for Web Modeler:
> With the Camunda **8.2.0** release and onward, **all** Camunda 8 Docker images are provided as multi-platform images.

Reply:
Thanks @jfriedenstab, now I understand your approach better here 🙂 

To not make this overly complicated, let's leave this as-is. If we need to increase the visibility or see confusion with our customers, we can look into adding a crisp statement to the announcements page.